src/components/post/PriceOfferModal/types.ts

```diff
@@ -1,10 +1,11 @@
 import type { ModalProps } from '@offer-ui/react'
+import type { TradeTypeCodes } from '@types'
 
 export type TradeAreaKeys = 'city' | 'county' | 'town'
 
 export type OfferForm = {
-  price?: number
-  tradeMethod?: number
+  price?: string
+  tradeType?: TradeTypeCodes
```

Comment:
```suggestion
  tradeTypeCode?: TradeTypeCodes
```
네이밍 타입에 맞게 수정되면 좋을 것 같습니다!

Reply:
흠 이렇게 되믄 tradetypeName을 내려주는 경우에도 대응이 되어야하는데 
너무 많은 부분에서 수정이 들어가야할 것 같아요 
의미 전달이 잘 되고 type으로 어떤 것이 전달되어야하는지 알 수 있으면 괜찮을 것 같은데 어떠신가융?

Reply:
좋십니다 좋십니다~~